eng/common/scripts/get-codeowners.ps1

```diff
@@ -1,7 +1,10 @@
 param (
   $TargetDirectory, # should be in relative form from root of repo. EG: sdk/servicebus
   $RootDirectory, # ideally $(Build.SourcesDirectory)
-  $VsoVariable = "" # target devops output variable
+  $AuthToken,
+  $OwningUsers = "", # target devops output variable
```

Comment:
I wouldn't have guess these are output variable names based on the names. We should name them so that is a little more obvious.